personio-personnel-data-api-oa3.yaml

```diff
@@ -2139,6 +2140,146 @@ paths:
             problem+json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+    post:
+      security:
+        - BearerAuth: [ ]
+      tags:
+        - Absences
+      description: Adds absence data for absence types **tracked in hours**.
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/CreateAbsencePeriodRequest"
+        description: Absence data
+        required: true
+      x-readme:
+        code-samples:
+          - language: "powershell"
+            code: "Auto-generated code snippets for this language may not be supported"
+          - language: "php"
+            code: "Auto-generated code snippets for this language may not be supported"
+      responses:
+        "201":
+          description: The absence period is successfully created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: "#/components/schemas/HourlyAbsencePeriodResponse"
+              examples:
+                response:
+                  value:
+                    success: true
+                    data:
+                      - type: AbsencePeriod
+                        attributes:
+                          id: 9bba303f-0fbc-4514-9958-0befa21923fb
```

Comment:
[Blocker]: The `id` field is listed here, but is not part of the `HourlyAbsencePeriodResponse`, nor the `HourlyAbsence` definition. Please add the missing field.